processing/src/main/java/org/apache/carbondata/processing/loading/sort/SortStepRowHandler.java

```diff
@@ -240,8 +258,44 @@ public IntermediateSortTempRow readWithNoSortFieldConvert(
     return new IntermediateSortTempRow(dictSortDims, noDictSortDims,measure);
   }
 
+  /**
+   * Read the data from the stream
+   *
+   * @param inputStream
+   * @param idx
+   * @return
+   * @throws IOException
+   */
+  private Object readDataFromStream(DataInputStream inputStream, int idx) throws IOException {
```

Comment:
is it refactor code ?? I think same code is also present for measures, can u please check and combine

Reply:
For measures, it will always be packed/unpacked to/from bytebuffer